app/src/main/res/values/strings.xml

```diff
@@ -25,6 +25,7 @@
     <string name="title_activity_login">Login</string>
     <string name="title_activity_accounts">Manage Accounts</string>
     <string name="title_activity_about">About</string>
+    <string name="title_activity_settings">Settings</string>
```

Comment:
we have now "Settings" two times in the strings.xml which will require effort for translating both. See how it is done for other activity titles, that are also used as navigation text (e.g. nav_about) to remove this duplication, but still allow to change it easily in future if needed

Reply:
Removed doubled setting